subgraph.goerli.yaml

```diff
@@ -143,4 +143,23 @@ dataSources:
           handler: handleTransferWithReferenceAndFee
           receipt: true
       file: ./src/erc20EscrowToPay.ts
-  
\ No newline at end of file
+  - kind: ethereum/contract
+    name: ERC20TransferrableReceivable
+    network: goerli
+    source:
+      abi: ERC20TransferrableReceivable
+      startBlock: 8444057
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.7
+      language: wasm/assemblyscript
+      entities:
+        - Payment
+      abis:
+        - name: ERC20TransferrableReceivable
+          file: ./abis/ERC20TransferrableReceivable.json
+      eventHandlers:
+        - event: TransferWithReferenceAndFee(address,address,uint256,indexed bytes,uint256,address)
```

Comment:
I'm confused why we're not indexing the `Payment` event emitted by the `payOwner` function in the `TransferableReceivable` contract.

Reply:
My confusion was resolved by this comment: https://github.com/RequestNetwork/requestNetwork/pull/1033#discussion_r1103717516